src/helpers/domainFindOptionsHelper.ts

```diff
@@ -0,0 +1,136 @@
+import { HttpRequest } from "@azure/functions";
+import { SortDirection } from "mongodb";
+import { DomainFindOptions } from "@zero-tech/data-store-core";
+import { DomainSortDirection, domainReflectionSchema } from "../types";
+declare type AscDesc = -1 | 1;
+declare type IncludeExlucde = 0 | 1;
+interface dynamicObject {
+  [key: string]: any;
+}
+const defaultProjection = ["domainid", "owner", "name"];
+const defaultSort = ["label", "owner"];
```

Comment:
While I think sorting by these props is useful. I do wonder if we should leave sorting decisions up to the consumers. As there is a performance cost to sorting. 

(I'm okay with this as is if there's a value we can pass to indicate no sorting)

Reply:
The consumer **are** allowed to decide how to sort. This is simply giving them a default sort if they dont specify one. The front end is always going to want to sort the results in some manner

Reply:
Is there a value that can be passed for users to indicate no sorting?

Reply:
Why would they want an unsorted response? Is that a valid use case?